### PR TITLE
Add Scaffolder Sentry Plugin

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
## Hey, I just made a Pull Request!

#15543 Adding a scaffolder Sentry Plugin.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  const { config } = options;
+
+  return createTemplateAction<{
+    organizationSlug: string;
+    teamSlug: string;
+    name: string;
+    slug: string;
+    authToken?: string;
+  }>({
+    id: 'sentry:create-project',
```
Comment (benjdlambert) [inline plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts:41]: I wonder if this should be something like `sentry:project:create` as we have `github:repo:create` already although no real strong opinion here to be honest.

File: plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+ * @public
+ */
+export function createSentryCreateProjectAction(options: { config: Config }) {
+  const { config } = options;
+
+  return createTemplateAction<{
+    organizationSlug: string;
+    teamSlug: string;
+    name: string;
+    slug: string;
```
Comment (benjdlambert) [inline plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts:38]:
```suggestion
    slug?: string;
```

File: plugins/scaffolder-backend-module-sentry/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@backstage/plugin-scaffolder-backend-module-sentry",
+  "version": "0.1.0",
```
Comment (benjdlambert) [inline plugins/scaffolder-backend-module-sentry/package.json:3]: I'd start this at `0.0.0` and then the changeset as minor will publish `0.1.0` :)

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Thanks for this! Looking pretty good, just some quick comments :pray:

File: plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts (hunk 1 above)
Comment (dpfaffenbauer) [reply plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts]: makes sense, also if we add more sentry scaffolder actions we have a clear namespaces for the actions.

is done.

File: plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts (hunk 2 above)
Comment (dpfaffenbauer) [reply plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts]: done

File: plugins/scaffolder-backend-module-sentry/package.json (hunk 3 above)
Comment (dpfaffenbauer) [reply plugins/scaffolder-backend-module-sentry/package.json]: done

**Comment by benjdlambert** [comment]
ah, you'll also want to run `yarn build:api-reports` again and commit the response.

**Comment by dpfaffenbauer** [comment]
@benjdlambert done

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Nice thanks @dpfaffenbauer!

**Comment by dpfaffenbauer** [comment]
🙌 🎉  from discovering backstage to my first contribution: 9 days.

**Pull request merged**